### ISSUE TITLE
Changing brightness via brightness applet doesn't work on boot

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#232]
mce restart is required to be able to change brightness. It's probably not on the right dbus bus?

**Comment by MerlijnWajer** [comment]
Figured this one out. `/etc/X11/Xsession.d/01dbus` unconditionally overrides any dbus info already written to `/tmp/`. `/etc/init.d/dbus-user` is started early on, and `mce` and `ke-recv` and others are on this dbus, but then anything in the X session is on the other user dbus. Simply commenting everything in `/etc/X11/Xsession.d/01dbus` and replacing it with a call to `/etc/osso-af-init/af-defines.sh` makes brightness and other stuff work again on boot.

**Comment by MerlijnWajer** [comment]
After above fix, psutils output also looks nice:

```
root@n900devuan:~# ps xua | grep dbus
message+  1741  0.3  0.9   4020  2276 ?        Ss   14:59   0:00 /usr/bin/dbus-daemon --system
user      1936  0.2  0.9   3776  2300 ?        S<s  14:59   0:00 /usr/bin/dbus-daemon --session --print-address=2
root      2832  0.0  0.6   3724  1540 pts/1    S+   15:02   0:00 grep dbus
```

Versus before:

```
root@n900devuan:~# ps xua | grep dbus
message+  1726  0.7  0.9   4028  2272 ?        Ss   14:37   0:00 /usr/bin/dbus-daemon --system
user      1921  0.1  0.8   3660  2136 ?        S<s  14:37   0:00 /usr/bin/dbus-daemon --session --print-address=2
user      2664  0.0  0.6   3676  1584 ?        S    14:38   0:00 dbus-launch --exit-with-session
user      2665  0.5  0.8   3792  2024 ?        Ss   14:38   0:00 /usr/bin/dbus-daemon --fork --print-pid 5 --print-address 7 --session
root      2822  0.0  0.6   3724  1572 pts/0    S+   14:38   0:00 grep dbus
```

**Comment by MerlijnWajer** [comment]
cc @freemangordon :D

**Comment by MerlijnWajer** [comment]
Should be fixed per `maemo-system-services` `0.5.3-0`.
